src/js/common/schemaform/FormApp.jsx

```diff
@@ -31,6 +31,9 @@ class FormApp extends React.Component {
       newProps.router.push(newProps.returnUrl);
       // Set loadedStatus in redux to not-attempted to not show the loading page
       newProps.setFetchFormStatus(LOAD_STATUSES.notAttempted);
+    } else if (newProps.loadedStatus === LOAD_STATUSES.prefillComplete) {
+      newProps.router.push(newProps.routes[newProps.routes.length - 1].pageList[1].path);
```

Comment:
So....I get that this is going to the "first" page in the form (after the intro), but how does that work? What does `newProps.routes` contain and why are we referencing the last element in there?

This question is probably long-overdue. 😳